lms/djangoapps/grades/management/commands/generate_persistent_grade.py

```diff
@@ -0,0 +1,41 @@
+"""
+Management command which update/generate persistent grades
+
+source ~/edxapp_env ; ./manage.py lms generate_persistent_grade --settings=production
```

Comment:
Add description what does it mean.
e.g. Usage:
Can be used like this:
or smth similar

Reply:
done